### ISSUE TITLE
About training with 8 gpus

### ISSUE THREAD

**Issue by NNNNAI** [TencentARC/GFPGAN#63]
Hi xintao, thanks for sharing your great work.
I currently trying to train GFPGAN with 8 gpus, which means the total batchsize will be double. Should I modified some hyperparameter in the train_gfpgan_v1.yml? Such as the learning rate and the totoal step, etc.  Thanks again, have a nice day~.

**Comment by xinntao** [comment]
1. I have also trained with 8 GPUs without modifying the hyper-parameters. The results are Ok with fewer iterations. 
But it may be not the optimal~

**Comment by NNNNAI** [comment]
Thanks for your help! Have a great day ~
